sendIntake.gs

```diff
@@ -397,7 +402,8 @@ const AppbuilderAPIPostRequest = ({ clients, ...intake }) => {
   }
 
   // case "Intake":
-  api.post.payload = JSON.stringify(data.Intake);
+  data.Intake["Feedback"] = data.Intake["Feedback"] ? data.Intake["Feedback"].replace(/[\u0800-\uFFFF]/g, c => `&#${c.charCodeAt(0)};`): data.Intake["Feedback"]
```

Comment:
`[\u0800-\uFFFF]/g` this will find all emojis, right? Not just the one in the dataset?

Reply:
Yes, it does